README.md

```diff
@@ -17,39 +17,32 @@ use atoum;
 
 class Example extends atoum
 {
-    protected $blackfire;
+    private $blackfireClient;
 
     public function testExemple()
     {
-        $config = new \Blackfire\Profile\Configuration();
-        $config
-            ->assert('main.wall_time < 2s', "Temps d'execution")
-        ;
-
-        $callback = function() {
-
-            sleep(4);
-
-            //some code and/or atoum assertions
-            $this->boolean(true)->isTrue();
-        };
-
         $this
-            ->blackfireProfile($this->getBlackfireClient(), $callback, $config)
-            ->matchesAssertions()
+            ->blackfire($this->getBlackfireClient())
+                ->assert('main.wall_time < 2s', "Temps d'execution")
+                ->profile(function() {
+                    sleep(4);
+                    //some code of atoum assertions
+                    $this->boolean(true)->isTrue();
+                })
```

Comment:
there is a setConfiguration method on the asserter,  to use a custom \Blackfire\Profile\Configuration but it's not "easy" to use.
i will add the  __call to a list of the Configuration methods, to ease defining those custom metrics.